pkg/xds/generator/testdata/ingress/meshgateway.envoy.golden.yaml

```diff
@@ -1,72 +1,4 @@
 resources:
-- name: backend
```

Comment:
Why are these no longer generated?

Which test does the best job of showing the changes of this PR?

Reply:
Good catch! That was a problem with the test itself. I think this test now shows PR changes the best because `04. several services in several meshes` used to generate only 1 `backend` cluster, but now it's `mesh-1:backend` and `mesh-2:backend`

Reply:
@michaelbeaumont is this explanation correct?  Can this conversation be resolved and this PR merged?